FWCore/SOA/interface/TableView.h

```diff
@@ -93,6 +93,16 @@ namespace edm {
       }
     };
 
+    template <typename T>
+    struct ViewFromTable;
+
+    template <typename... T>
+    struct ViewFromTable<Table<T...>> {
+      using type = TableView<T...>;
+    };
+    template <typename T>
+    using ViewFromTable_t = typename ViewFromTable<T>::type;
+
```

Comment:
Question, did you check to be sure this syntax works? I was just doing it off the top of my head :P

Reply:
Yes, thanks again! I'm using it for all the SOA table view typedefs now so it works well! I just needed to ad the `typename` to your suggestion.